src/types/types.ts

```diff
@@ -10,14 +10,21 @@ export type IPCFile = {
 	path: string;
 	permission: IPCPermission;
 	deletedAt: number | null;
+	logs: FileLogs[];
 };
 
 export type IPCFolder = {
 	name: string;
 	createdAt: number;
 	path: string;
+	logs: FileLogs[]
 };
 
+export type FileLogs = {
```

Comment:
```suggestion
export type FileLog = {
```
And replace its usage too :wink: